### PR TITLE
Update test

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- Test system modified
- `is_same_data` function modified

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri Please review this PR 🚀 
If everything is okay, **Squash/Merge** it after approval 💯

File: test/overall_test.py [hunk 1]
```diff
@@ -208,6 +208,22 @@
...
+>>> g2 = GenerativeImage(function1=function1, function2=function2)
+>>> g1.generate(seed=22)
+>>> g2.generate(seed=22)
+>>> is_same_data(g1.data1, g2.data1)
+True
+>>> is_same_data(g1.data2, g2.data2)
+True
+>>> len(g1.data1)>0
+True
+>>> len(g1.data2)>0
```
Comment (sadrasabouri) [inline test/overall_test.py:223]:
```
>>> len(g1.data1) > 0
True
>>> len(g1.data2) > 0
```

File: test/overall_test.py [hunk 2]
```diff
@@ -208,6 +208,22 @@
 >>> del(g)
 >>> del g_.data1
 >>> del(g_)
+>>> g1 = GenerativeImage()
+>>> function1 = eval("lambda x,y:" + g1.function1_str)
+>>> function2 = eval("lambda x,y:" + g1.function2_str)
```
Comment (sadrasabouri) [inline test/overall_test.py:213]:
```
>>> function1 = eval("lambda x, y:" + g1.function1_str)
>>> function2 = eval("lambda x, y:" + g1.function2_str)
```

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Minor edit related to spacing in tests.

File: test/overall_test.py (hunk 2 above)
Comment (sepandhaghighi) [reply test/overall_test.py]: fixed in https://github.com/sepandhaghighi/samila/pull/126/commits/c9b14e1ac8f8533be3917348d7105e3cedd9a19f

File: test/overall_test.py (hunk 1 above)
Comment (sepandhaghighi) [reply test/overall_test.py]: fixed in https://github.com/sepandhaghighi/samila/pull/126/commits/c9b14e1ac8f8533be3917348d7105e3cedd9a19f

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**